change/@fluentui-react-dialog-bbb01d69-5227-40c3-aaf3-053f1af35f0d.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "chore: updates dialog stories",
```

```suggestion
  "comment": "chore: cleanups in types",
```

_nit:_ We add into changelogs only changes that affects published code.